sys/kern/cred.c

```diff
@@ -7,3 +7,8 @@ void cred_fork(proc_t *to, proc_t *from) {
   assert(mtx_owned(&from->p_lock));
   memcpy(&to->p_cred, &from->p_cred, sizeof(cred_t));
 }
+
+void cred_copy(cred_t *cr, proc_t *p) {
+  assert(mtx_owned(&p->p_lock));
+  memcpy(cr, &p->p_cred, sizeof(cred_t));
+}
```

Comment:
I don't think we need this anymore, do we?

Reply:
Yes. Removed.

Reply:
It hasn't been removed here. Have you pushed your branch?